specs/externalLinks/external_links_api_specs.yaml

```diff
@@ -0,0 +1,175 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: External links create api
+paths:
+  /orchestrator/global/externalLinks:
+    post:
+      description: Save newly created linkout 
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateExternalLinks'
+            
+      responses:
+        "200":
+          type: boolean
+          description: True- Successfully created external link, false- error in saving
+
+  /orchestrator/global/externalLinks:
+    get:
+      description: Get all available external links for all clusters 
+      requestBody:
+        required: false
+      
+      responses:
+        "200":
+          description: Successfully fetched external links
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetAllExternalLinks"
```

Comment:
Object name can not be "GetAllExternalLinks".

It should be ExternalLinks (Which is an array of ExternalLink)